src/test/java/com/ays/util/AysMockResultMatchersBuilders.java

```diff
@@ -31,4 +31,24 @@ public static JsonPathResultMatchers response() {
         return MockMvcResultMatchers.jsonPath("$.response");
     }
 
+    public static JsonPathResultMatchers responseAccessToken() {
+        return MockMvcResultMatchers.jsonPath("$.response.accessToken");
+    }
+
+    public static JsonPathResultMatchers responseAccessTokenExpiresAt() {
+        return MockMvcResultMatchers.jsonPath("$.response.accessTokenExpiresAt");
+    }
+
+    public static JsonPathResultMatchers responseRefreshToken() {
+        return MockMvcResultMatchers.jsonPath("$.response.refreshToken");
+    }
+
+    public static JsonPathResultMatchers responseUsername() {
+        return MockMvcResultMatchers.jsonPath("$.response.username");
+    }
+
+    public static JsonPathResultMatchers responsePassword() {
+        return MockMvcResultMatchers.jsonPath("$.response.password");
+    }
+
 }
```

Comment:
If the improvements mentioned in my previous comments are implemented, there should be no need to add these fields here. These fields are very specific and not used in every test, so it wouldn't be appropriate to include them here.

Reply:
I just removed a relevant part from `AysMockResultMatchersBuilders`. Can we look through it again?

Reply:
@Rapter1990, There shouldn't have been any additions made to this section after the development, but I still see some methods added. Could you please review it again?